server/src/tasks.clj

```diff
@@ -89,5 +99,6 @@
   "Helper to setup everything the server needs for its initial run."
   [_args]
   (ensure-override-config)
+  (copy-hooks)
```

Comment:
I don't know if we can count on oss people having clj-kondo installed. Could we just make it opt-in for everybody? Personally, I'd rather the test fail than have to wait for clj-kondo on every commit.

Reply:
Made it optional via `make install-hooks`. OSS people can just rely on our CI

Not sure what you mean by “waiting on clj-kondo”, it takes like 50ms (it’s only run on changed files)